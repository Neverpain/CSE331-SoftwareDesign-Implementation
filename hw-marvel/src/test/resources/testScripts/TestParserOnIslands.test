# Tests the makeGraph method on the islands file to see that it is
# built correctly, prints out nodes and children of nodes(no children
# since there are no connections)

LoadGraph g islands.csv

ListNodes g
ListChildren g Victoria
ListChildren g Java
ListChildren g Honshu
ListChildren g Borneo