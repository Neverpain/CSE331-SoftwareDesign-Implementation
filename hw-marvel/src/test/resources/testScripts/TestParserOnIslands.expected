# Tests the makeGraph method on the islands file to see that it is
# built correctly, prints out nodes and children of nodes(no children
# since there are no connections)

loaded graph g

g contains: Baffin Borneo Ellesmere Greenland Honshu Java Madagascar Sumatra Victoria
the children of Victoria in g are:
the children of Java in g are:
the children of Honshu in g are:
the children of Borneo in g are: